scripts/CMakeLists.txt

```diff
@@ -65,47 +64,48 @@ set(SCRIPTS
 	TrainingMission
 	Y2K)
 
+add_custom_target(HogLinuxFull-copy
```

Comment:
if (UNIX)

Reply:
HogLinuxFull-copy is dependency of SCRIPTS targets so it can't be hided behind UNIX guard. Copying of directory is not a big deal for cross-platform support. I think, next step would be recompile hog file for every platform including Windows.